precompile/contracts/nativeminter/nativemintertest/compile.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package nativemintertest
+
+// Step 1: Compile Solidity contracts to generate ABI and bin files
+//go:generate solc-v0.8.30 -o artifacts --overwrite --abi --bin --base-path ../../../.. precompile/=precompile/ --evm-version paris NativeMinterTest.sol
```

Comment:
we should specify cancun
```suggestion
//go:generate solc-v0.8.30 --evm-version cancun -o artifacts --overwrite --abi --bin --base-path ../../../.. precompile/=precompile/ NativeMinterTest.sol
```